doc/src/modules/printing.rst

```diff
@@ -307,31 +307,53 @@ This class implements Python printing. Usage::
     x = Symbol('x')
     e = 5*x**3 + sin(x)
 
-srepr
------
+Srepr
+-----------
 
 .. module:: sympy.printing.repr
 
-This printer generates executable code. This code satisfies the identity
-``eval(srepr(expr)) == expr``.
+SymPy doesn’t use repr() for generating textual representation of expressions.
 
-``srepr()`` gives more low level textual output than ``repr()``
+Usage::
 
-Example::
+    >>> repr(5*x**3 + sin(x))
+    '       5*x**3 + sin(x)'
```

Comment:
I followed this [examples code](https://github.com/sympy/sympy/blob/master/doc/src/modules/printing.rst#fortran-printing) ,when I was getting checks failed.